test/samples/nullable/input.json

```diff
@@ -1,16 +1,20 @@
 {
-	"id": "http://yourdomain.com/schemas/myschema.json",
-	"description": "Example description",
-	"type": "object",
-	"properties": {
-		"first_name": {
-			"type": "string"
-		},
-		"last_name": {
-			"type": "string"
-		}
-	},
-	"required": [
-		"first_name"
-	]
+    "id": "http://yourdomain.com/schemas/myschema.json",
+    "description": "Example description",
+    "allOf": [
```

Comment:
We dont need to add allOf here as we have other examples that do this, can we revert this please.